src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu.style.ts

```diff
@@ -0,0 +1,70 @@
+import styled, { css } from "styled-components";
+
+import Button from "../../button";
+
+interface StyledResponsiveMenuProps {
+  height?: string;
+  left?: string;
+  reduceMotion?: boolean;
```

Comment:
As mentioned above I don't think this will ever not be `true/false` so we could remove `?`